viser/client/src/SceneTree.tsx

```diff
@@ -245,12 +255,45 @@ export function SceneNodeThreeObject(props: {
     <>
       {React.useMemo(() => makeObject(setRef), [makeObject, setRef])}
       {obj !== null && (
-        <SceneNodeThreeChildren
-          name={props.name}
-          useSceneTree={props.useSceneTree}
-          parent={obj}
-        />
+        <group>
+          <SceneNodeLabel 
+            visible={visibility && labelVisibility}
+            text={props.name} 
+          />
+          <SceneNodeThreeChildren
+            name={props.name}
+            useSceneTree={props.useSceneTree}
+            parent={obj}
+          />
+        </group>
       )}
     </>
   );
 }
+
+export type SceneNodeLabelProps = {
```

Comment:
it seems like these exports are unnecessary

Reply:
Removed export